src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/model/RepoInfoWithStatus.java

```diff
@@ -81,11 +86,11 @@ public String getRemote() {
     return remote;
   }
 
-  public boolean isRemote() {
```

Comment:
Why has this method been removed?

Reply:
As this is now just a JSON wrapper, no-one is using the isRemote method.